executor/aggfuncs/aggfunc_test.go

```diff
@@ -263,6 +264,61 @@ func rowMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType) (memDeltas
 	return memDeltas, nil
 }
 
+type multiArgsUpdateMemDeltaGens func(*chunk.Chunk, []*types.FieldType) (memDeltas []int64, err error)
+
+func defaultMultiArgsMemDeltaGens(srcChk *chunk.Chunk, dataTypes []*types.FieldType) (memDeltas []int64, err error) {
+	memDeltas = make([]int64, 0)
+	m := make(map[string]bool)
+	for i := 0; i < srcChk.NumRows(); i++ {
+		row := srcChk.GetRow(i)
+		if row.IsNull(0) {
+			memDeltas = append(memDeltas, int64(0))
+			continue
+		}
+		memDelta := int64(0)
+		val := ""
+		memValDelta := int64(0)
+		memKeyDelta := int64(0)
+		for j, dataType := range dataTypes {
```

Comment:
Can we handle key and value separately? The length of the key can be obtained directly through `len(key.ToString())`, and `partialResult4JsonObjectAgg` has only two parameters, it is not appropriate to use range

Reply:
But how can be get the key from the chunk without iterate over the dataTypes?